lib/rules/no-regex-spaces.js

```diff
@@ -37,11 +37,11 @@ module.exports = {
          * @private
          */
         function checkRegex(node, value, valueStart) {
-            const multipleSpacesRegex = /( {2,})+?/,
+            const multipleSpacesRegex = /( {2,})+?( [+*{?]|[^+*{?])/,
```

Comment:
I think this will give the wrong result if the spaces are at the end of the regex. For example, if the regex contains five spaces at the end, only four spaces will be included in the first match (the last space will be included in the second match).

Setting `multipleSpacesRegex` to `/( {2,}?)( [+*{?]|[^ ]|$)/` might fix the issue.